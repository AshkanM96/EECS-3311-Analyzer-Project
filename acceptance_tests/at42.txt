conjoin
integer_constant(1)
integer_constant(5)
type_check
evaluate
reset
conjoin
boolean_constant(true)
boolean_constant(true)
type_check
evaluate
reset
conjoin
boolean_constant(true)
disjoin
boolean_constant(true)
boolean_constant(false)
type_check
evaluate
