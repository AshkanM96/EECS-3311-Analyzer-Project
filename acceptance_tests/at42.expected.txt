  Expression currently specified: ?
  Report: Expression is initialized.
->conjoin
  Expression currently specified: (? && nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: (1 && ?)
  Report: OK.
->integer_constant(5)
  Expression currently specified: (1 && 5)
  Report: OK.
->type_check
  Expression currently specified: (1 && 5)
  Report: (1 && 5) is not type-correct.
->evaluate
  Expression currently specified: (1 && 5)
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->conjoin
  Expression currently specified: (? && nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True && ?)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True && True)
  Report: OK.
->type_check
  Expression currently specified: (True && True)
  Report: (True && True) is type-correct.
->evaluate
  Expression currently specified: (True && True)
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->conjoin
  Expression currently specified: (? && nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True && ?)
  Report: OK.
->disjoin
  Expression currently specified: (True && (? || nil))
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True && (True || ?))
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (True && (True || False))
  Report: OK.
->type_check
  Expression currently specified: (True && (True || False))
  Report: (True && (True || False)) is type-correct.
->evaluate
  Expression currently specified: (True && (True || False))
  Report: True
